src/Prefixer.js

```diff
@@ -82,12 +98,20 @@ export function prefix (value, length) {
 						return ~indexof(value, 'stretch') ? prefix(replace(value, 'stretch', 'fill-available'), length) + value : value
 				}
 			break
+		// grid-(column|row)
+		case 5152: case 5920:
+			// simple position value
+			if (match(substr(value, length + 1, sizeof(value) - 1), /^\d+$/)) {
+				return MS + value + value
+			} else if (match(substr(value, length + 1, sizeof(value) - 1), /(\d)+\s*\/( ?span)?\s*(\d)+/)) {
+				return replace(substr(value, length + 1, sizeof(value) - 1), /(\d)+\s*\/( ?span)?\s*(\d)+/, (_, $1, $span, $3) => MS + substr(value, 0, length + 1) + $1 + ';' + MS + substr(value, 0, length) + '-span:' + ($span ? $3 : +$3 - +$1) + ';') + value
```

Comment:
Will thoroughly review tomorrow evening.

Reply:
I've refactored this part in https://github.com/thysultan/stylis/pull/285/commits/6adc047a7cef8db8a613b684e873131bacfd4a43